scripts/dts/edtlib.py

```diff
@@ -768,13 +768,23 @@ def _init_binding(self):
             on_bus = self.on_bus
 
             for compat in self.compats:
+                # When matching, respect the order of the 'compatible' entries,
+                # and for each one first try to match against an explicitly
+                # specified bus (if any) and then against any bus. This is so
+                # that matching against bindings which do not specify a bus
+                # (i.e. all Zephyr-external ones, since this is a Zephyr
```

Comment:
nit: not sure what the (ie..) part of the comment is trying to convey.  As at some point I'm guessing in zephyr we'll have bindings that utilize this functionality.

Reply:
True. I think "e.g." would be more accurate. I can still change it if you want.

Reply:
I removed the entire comment and added a similar mention to the commit message